client/src/lib/paymentCalc.ts

```diff
@@ -92,16 +96,15 @@ export async function calculateFeeDataExtended(
     // overrideAxiomQueryFeeL2 = AXIOM_QUERY_FEE + projectedCallbackCost - maxFeePerGas * (callbackGasLimit + proofVerificationGas)
     const overrideAxiomQueryFeeL2 = axiomQueryFee + projectedCallbackCost - maxFeePerGas * (callbackGasLimit + proofVerificationGas);
```

Comment:
AXIOM_QUERY_FEE here should be the hard-coded default value, not read from contract.